process-engine/process_engine_runtime
# ✨Add onProcessError notification

## Changes

1. Add `onProcessError` notification
2. Run `npm ci` instead of `npm install` during Jenkins builds, if the build runs on the `master` or `develop` branch

## Issues

Closes #348 

PR: #355

## How to test the changes

- Subscribe for a ProcessError notification, using the `onProcessError` UseCase exposed by the Management API and Consumer API
- Run a Process that causes an error (easiest would be with a dummy script task that uses invalid script code)
- See that you'll get the notification
